assertions.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "method": "GET",
+    "path": "/foo",
+    "expect": {
+      "upstream_host": "foo.local:3001",
+      "upstream_path": "/foo",
+      "upstream_method": "GET"
+    }
+  },
+  {
+    "method": "GET",
+    "path": "/bar",
+    "expect": {
+      "upstream_host": "bar.local",
+      "upstream_path": "/bar",
+      "upstream_method": "GET"
+    }
```

Comment:
Todo: this is an https route. We should be able to assert `"upstream_host": "bar.local:443"` but that fails. Need to igure out why and fix it.